frontend/src/bundles/common/components/user-photo-uploader/user-photo-uploader.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react';
+import { IconContext } from 'react-icons';
+import { GrGallery } from 'react-icons/gr';
+
+import styles from './styles.module.scss';
+
+interface UploadProperties {
+    setImage: React.Dispatch<React.SetStateAction<string | ArrayBuffer | null>>
+}
+
+const UserPhotoUploader: React.FC<UploadProperties> = ({ setImage }) => {
+
+    const handleFile = useCallback((file: File) => {
+        const reader = new FileReader();
+        reader.addEventListener('load', () => {
+            if (reader.readyState === 2) {
```

Comment:
2 looks like a magic number. 
Create a const for it

Reply:
Done